src/Encamina.Enmarcha.SemanticKernel/Extensions/KernelExtensions.cs

```diff
@@ -228,44 +283,34 @@ public static IEnumerable<KernelPlugin> ImportPromptFunctionsFromAssembly(this K
     private static int GetMaxTokensFromKernelFunction(Kernel kernel, KernelFunction function, KernelArguments arguments)
     {
         // Try to use IChatCompletionService as the IAService to retrieve the service settings, but fallback to ITextGenerationService if it's not available.
-        // Once the service settings are retrieved, get the value of the `max_tokens` property from the extension data.
+        // Once the service settings are retrieved, get the value of the MaxTokens property from the MaxTokens property or extension data.
         // If the `max_tokens` property is found, check is a JsonElement and try to get its value as an integer.
         // Finally, if the value is an integer, return it.
-        // In any other case (if the service settings are not found, or the `max_tokens` property is not found, or the value is not an integer), throw an exception.
```

Comment:
In the comment it is said: "In any other case (if the service settings are not found, or the `max_tokens` property is not found, or the value is not an integer), throw an exception." yet in the method no exception is throwed, maybe the comment is outdated?

Reply:
Fixed, well spotted!